xbmc/addons/AddonManager.cpp

```diff
@@ -607,6 +607,8 @@ bool CAddonMgr::FindAddon(const std::string& addonId,
   FindAddons(installedAddons, "special://xbmcbin/addons");
   FindAddons(installedAddons, "special://xbmc/addons");
   FindAddons(installedAddons, "special://home/addons");
+  FindAddons(installedAddons, "special://xbmcbinaddons");
+  FindAddons(installedAddons, "special://xbmcaltbinaddons");
```

Comment:
and `special://xbmcaltbinaddons` is however not mapped to anything on my linux build.
(your description suggests you're using a unix-like os, too)